janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/JanusGraphElasticsearchContainer.java

```diff
@@ -80,4 +93,15 @@ public ModifiableConfiguration setConfiguration(ModifiableConfiguration config,
         config.set(BULK_REFRESH, "wait_for", index);
         return config;
     }
+
+    private void create_default_index_template() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'create_default_index_template' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4179483)